spec/support/jasmine-browser.json

```diff
@@ -0,0 +1,23 @@
+{
+  "srcDir": "public/assets",
+  "srcFiles": [
+    "application-*.js"
+  ],
+  "cssFiles": [
+    "application-*.css"
+  ],
+  "specDir": "spec/javascripts",
+  "specFiles": [
+    "**/*[sS]pec.js"
+  ],
+  "helpers": [
+    "helpers/*.js",
+    "vendor/*.js"
```

Comment:
If we do use vendors for deps I imagine it might need to come first in this list (I wonder why they don't have it as first in their examples)